kotsadm/web/src/components/apps/AppVersionHistory.jsx

```diff
@@ -61,7 +61,8 @@ class AppVersionHistory extends Component {
     showUpdateCheckerModal: false,
     displayShowDetailsModal: false,
     yamlErrorDetails: [],
-    deployView: false
+    deployView: false,
+    versionSequence: ""
```

Comment:
hmmm, i feel like there can be a better name for this. this does not describe what this variable is used for IMO.